Data/Map/Lazy.hs

```diff
@@ -88,24 +88,28 @@ module Data.Map.Lazy (
     -- * Map type
     Map              -- instance Eq,Show,Read
 
+    -- * Construction
+    , empty
+    , singleton
+    , fromList
+    , fromListWith
+    , fromListWithKey
+
     -- * Operators
-    , (!), (!?), (\\)
 
     -- * Query
-    , null
-    , size
-    , member
-    , notMember
     , lookup
+    , (!?)
     , findWithDefault
+    , member
+    , notMember
     , lookupLT
     , lookupGT
     , lookupLE
     , lookupGE
-
-    -- * Construction
-    , empty
-    , singleton
+    , null
+    , size
+    , (!)
```

Comment:
I don't like partial functions, but `!`really belongs in the lookup zone.

Reply:
Wait, isn't that where it is? The construction heading moved elsewhere so it's under the query heading. The interleaved diffs are confusing sometimes. Unless I've completely misinterpreted your comment :P

Reply:
I want `(!)` adjacent to `(!?)`.

Reply:
Done.